modules/badge/css/README.md

```diff
@@ -0,0 +1,74 @@
+# Canvas Kit CSS Badge
+
+The count badge provides a quantity-based summary with dynamic values.
+
+## Installation
+
+```sh
+yarn add @workday/canvas-kit-css
+```
+
+or
+
+```sh
+yarn add @workday/canvas-kit-css-badge
+```
+
+Add your `node_modules` directory to your SASS `includePaths`. You will then be able to import
+`index.scss`.
+
+```scss
+@import '~@workday/canvas-kit-css-badge/index.scss';
+```
+
+## Accessibility
+
+A common use case for the count badge is for displaying notifications, and there are several
+accessibility concerns you'll want to keep in mind:
+
+- The button should have an aria-label that updates with the count
+- The elements inside the button should have `aria-hidden`
+- The live region should be outside the button
+- The live region should be visually hidden and only contain text
+
+### Example
+
+```html
+<button type="button" aria-label="Alerts 3 new notifications">
+  <svg aria-hidden="true" focusable="false">...</svg>
+  <span class="wdc-count-badge" aria-hidden="true">3</span>
+</button>
+<div class="wdc-accessible-hide" role="status" aria-live="polite" aria-atomic="true">
```

Comment:
Is the class called `wdc-accessible-hide`? This is supposed to be only visually hidden, right?

Reply:
Correct. That's the class specified here:
https://github.com/Workday/canvas-kit/blob/master/modules/common/css/lib/accessibility.scss#L38-L40

I assume they would import the common utils / classes, but maybe not. I can specify that.

Reply:
Probably easier to just change to `css={accessibleHide}`

Reply:
Probably easier to just change to `css={accessibleHide}`

Reply:
Updated! 👍